### DISCUSSION TITLE
Another colors thread

### DISCUSSION THREAD

**Discussion by end-4** [end-4/dots-hyprland#578] [Ideas]
guys @midn8hustlr look chrome :thinking:

https://github.com/end-4/dots-hyprland/assets/97237370/f3d7cc42-3398-44d5-94be-8c2343a24248

i think for the terminal they just use background and onBackground, then keep the other colors
i don't say let's do this, i'm just sharing this, wondering if it's a good idea

**Comment by midn8hustlr** [comment]
So, this means changing the term0 and term15 colors only and leaving everything as it is?
I feel this would make the terminal scheme very similar between the themes, and less enjoyable. You could achieve something similar just by changing those harmonization arguments while executing the python script.
At the end, this is basically a tradeoff between productivity and eye-candy. I believe preserving the terminal scheme is more practical from the usability perspective. For eg. a wrong command you can quickly identify as color red, while using a scheme you have to train your eyes to map to a different color. But this snatches away the beautification, cause hey, this is Hyprland, and most of us people are here just for the eyecandy.

So that's why I have kept things adjustable, by changing those parameters and thresholds you can control how much of the original colors you want to preserve. Maybe we can add some slider to easily control this behaviour via sidebar (which I was planning to implement anyways and many other customization stuffs too, but got quite busy in last few weeks).

Regarding the term0 and term15, I could have choosen 'background' and 'onBackground', but they lacked vibrance (not much distinguishable), so instead I went for the 'surface' colors. Even with surface colors, I had to boost the chroma and tone to make it more vibrant.

I believe Material You does a very good job in tailoring the system according to your taste, which is closer to You (atleast to your wallpaper). But Google still haven't figured out how to decorate the code colorscheme using Material theme, i guess cause most programmers (apart from customization freaks like you or me) actually don't care.

So, I thought using this harmonization for terminal schemes would be a nice experiment to keep a balance between usability and eye-candy, and I am pretty much satisfied with it. I still have to figure out some other things like how to do color schemes on text editor (I use vim), and I have a custom Material Tmux theme too (which i will try to release sometime).

**Reply by end-4** [reply]
well yea what i kinda miss is the red color for wrong commands
i'm lazy so i'll leave things as is for now

**Reply by midn8hustlr** [reply]
@end-4 you only made it less-reddish after you changed the default harmonization threshold ¯\_(ツ)_/¯
